### PR TITLE
gem dump

### PR DESCRIPTION
**Pull request by mensfeld**
@agwozdowski please review
@pavlo-vavruk  please review

### CODE REVIEW

File: lib/karafka/base_controller.rb [hunk 1]
```diff
@@ -60,8 +60,7 @@ class BaseController
     # We won't run perform at the backend if any of the callbacks
     # returns false
     # @see http://api.rubyonrails.org/classes/ActiveSupport/Callbacks/ClassMethods.html#method-i-get_callbacks
-    define_callbacks :schedule,
-      terminator: ->(_target, result) { result == false }
+    define_callbacks :schedule
```
Comment (pavlo-vavruk) [inline lib/karafka/base_controller.rb]: comments above becomes outdated.

Comment (mensfeld) [inline lib/karafka/base_controller.rb]: updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +15,8 @@
 - #82 - Karafka autodiscovery fails upon caching of configs
 - #81 - Switch config management to dry configurable
 - Version fix
+- Dropped support for Ruby 2.1.*
+- Ruby dump
```
Comment (pavlo-vavruk) [inline CHANGELOG.md]: Maybe add details about version here? It can be useful.

Comment (mensfeld) [inline CHANGELOG.md]: true, added

**Comment by pavlo-vavruk** [comment]
👍  it looks good to me

**Comment by agwozdowski** [comment]
👍

**Pull request merged**